src/transformers/tokenization_utils.py

```diff
@@ -1608,8 +1748,5 @@ def decode(self, token_ids, skip_special_tokens=False, clean_up_tokenization_spa
         else:
             return text
 
-    def decode_batch(self, ids_batch, skip_special_tokens=False, clear_up_tokenization_spaces=True):
-        return [
-            self.clean_up_tokenization(text) if clear_up_tokenization_spaces else text
-            for text in self.tokenizer.decode_batch(ids_batch, skip_special_tokens)
-        ]
+    def save_vocabulary(self, save_directory):
+        self._tokenizer.save(save_directory)
```

Comment:
`save` expects a `directory` and a `name`.

Reply:
Fixed in 72e0a05